design/one-pager-composition-revisions.md

```diff
@@ -116,9 +116,9 @@ spec:
 apiVersion: apiextensions.crossplane.io/v1alpha1
 kind: CompositionRevision
 metadata:
-  # The revision's name is derived by appending the revision number to the
-  # Composition's name.
-  name: example-1
+  # The revision's name is derived from the composition's name using the API
+  # server's GenerateName support.
+  name: example-fklem2
```

Comment:
Was there discussion around this change? I seem to remember someone mentioning it would be nice to be able to easily identify the revision number, but I am in support of this change because revision is in the spec and, more importantly for UX, defined as a printer column 👍🏻

Reply:
The original discussion was in https://github.com/crossplane/crossplane/pull/2437#discussion_r679028275.

I could have sworn I made this change in a distinct commit with the rationale, but apparently not. I ended up going back to this approach because I realised it was more consistent with what we do for `ProviderRevision` and `ConfigurationRevision`. I like the idea of including the revision in the name, but I prefer consistency and as you say we show the revision pretty prominently as a printer column.